src/www/ui/api/Models/Analysis.php

```diff
@@ -103,14 +103,14 @@ public function __construct($bucket = false, $copyright = false, $ecc = false, $
    * @param array $analysisArray Associative boolean array
    * @return Analysis Current object
    */
-  public function setUsingArray($analysisArray)
+  public function setUsingArray($analysisArray, $version = ApiVersion::V1)
   {
     if (array_key_exists("bucket", $analysisArray)) {
       $this->bucket = filter_var($analysisArray["bucket"],
         FILTER_VALIDATE_BOOLEAN);
     }
-    if (array_key_exists("copyright_email_author", $analysisArray)) {
-      $this->copyright = filter_var($analysisArray["copyright_email_author"],
+    if (array_key_exists(($version == ApiVersion::V1? "copyright_email_author" : "copyrightEmailAuthor"), $analysisArray)) {
```

Comment:
Could we reverse the logic, the V1 should be the default as we're normally doing in controllers.
 
`($version == ApiVersion::V2 ? "copyrightEmailAuthor":"copyright_email_author")`

Reply:
Done